pages/archiving/setup.rst

```diff
@@ -56,6 +56,127 @@ The archived indices will be stored in a backend. A backend that stores the data
 when the server starts for the first time but you can create a new backend if you want to store the data in a different
 path.
 
+
+S3 Archiving
+~~~~~~~~~~~~
+
+Graylog provides archiving built to be used against AWS S3. This doesn't mean it's limited
+to the AWS solution. You can configure S3 compatible solutions like MinIO, CEPH, Digital Ocean 
+Spaces and others.
+
+On the *Archive* page:
+
+#. Click the *Manage Backends* button on the top right.
+#. Click *Create Backend* under the *Archive Backends*; this takes you to *Edit archive backend configuration options*.
+#. Go to the *Backend configuration* section and on the *Backend Type* dropdown select *S3*.
+#. Fill out the form, completing the fields that best suit your choice of archive.
+
++-----------------------------+-------------------------------------------+
+| Name                        | Description                               | 
++=============================+===========================================+
+| Title                       | A simple title to identify the backend    |     
++-----------------------------+-------------------------------------------+
+| Description                 | Longer description for the backend        |     
++-----------------------------+-------------------------------------------+                                                
+| S3 Endpoint URL             | Only configure this if not using AWS      |                                                
++-----------------------------+-------------------------------------------+
+| AWS Authentication Type     | Choose access type from the dropdown menu |     
++-----------------------------+-------------------------------------------+                                                 
+| AWS Assume Role (ARN)       | This is an optional input for             |
+|                             | alternate authentication mechanisms.      |      
++-----------------------------+-------------------------------------------+     
+| Bucket Name                 | The name of the S3 bucket                 |     
++-----------------------------+-------------------------------------------+     
+| Spool Directory             | Temporary name for the archiving process  |    
```

Comment:
Directory where archiving data is stored before being uploaded

Reply:
Updated